examples/dash/dash-live-updates/README.md

```diff
@@ -0,0 +1,44 @@
+# Real-Time Bitcoin Tracker with Dash and WebSocket API 
+
+Explore how to display real-time, time-series data using Dash and a PostgreSQL extension, TimescaleDB, to dynamically update Bitcoin prices and aggregate trades fetched via the Binance WebSocket API.
+
+![Dash App](dashboard.png)
+
+## Steps for Testing Locally
+
+### 1. Development Environment Setup
+```sh
+conda create --name YOUR_ENV_NAME python=3.11
+conda activate YOUR_ENV_NAME
+pip install -r requirements.txt
+```
+
+### 2. Save Database Connection Details as Environment Variables
+```
+DB_USER=<USERNAME>
+DB_PASSWORD=<PASSWORD>
+DB_NAME=<DATABASE_NAME>
+DB_HOST=<HOST>
+DB_PORT=<PORT>
+```
+
+### 3. Backend Setup
```

Comment:
Add steps for setting up TimescaleDB with screenshots.

Reply:
[fixed](https://github.com/ploomber/doc/pull/269/files#diff-cd1a6fcc32f127145ab61284c47ea54083a31b43a7ef552054afbdee033a3d83R27-R49)